Airbnb.Infrastructure/KafkaSender/EventSender.cs

```diff
@@ -0,0 +1,18 @@
+using Airbnb.Application.Interfaces;
+
+namespace Airbnb.Infrastructure.KafkaSender;
+
+public class EventSender : IEventSender
+{
+    private readonly IKafkaProducer _producer;
+
+    public EventSender(IKafkaProducer producer)
+    {
+        _producer = producer;
+    }
+
+    public async Task SendEvent<T>(string topic, string key, T jsonMessage)
+    {
+        await _producer.ProduceAsync(topic, key, jsonMessage); 
+    }
```

Comment:
key is not always a string but an object. And you probably should serialize this object, because you can have complex keys with several fields